### ISSUE TITLE
New sr implementation broke python 3.4 compatibility

### ISSUE THREAD

**Issue by benlapETS** [MetPX/sarracenia#268]
Uses of pathlib PosixPath read_text broke 3.4 version

```
gathering global state: procs, configs, Traceback (most recent call last):
  File "/local/ben/anaconda3/envs/base_sarra34/bin/sr", line 10, in <module>
    sys.exit(main())
  File "/local/ben/git/sarracenia/sarra/sr.py", line 608, in main
    gs = sr_GlobalState()   
  File "/local/ben/git/sarracenia/sarra/sr.py", line 350, in __init__
    self._read_states() 
  File "/local/ben/git/sarracenia/sarra/sr.py", line 193, in _read_states
    t = f_obj.read_text().strip()
AttributeError: 'PosixPath' object has no attribute 'read_text'
```

This will be an easy fix

**Comment by benlapETS** [comment]
The fix has been commit and merged to master with #269

**Comment by petersilva** [comment]
and the fix is in the latest version... and it works.
